Libraries/Components/Navigation/NavigatorIOS.ios.js

```diff
@@ -679,14 +691,19 @@ var NavigatorIOS = React.createClass({
     // computation of navigator children.
     var items = shouldRecurseToNavigator ?
       this.state.routeStack.map(this._routeToStackItem) : null;
+    var optionalProps = {};
+    if ('interactivePopGestureEnabled' in this.props) {
```

Comment:
We don't typically use this pattern. You can just pass this directly to the NavigatorComponent. If it's not configured the value of `this.props.interactivePopGestureEnabled` will be undefined and React will handle that properly.